spec/models/person_spec.rb

```diff
@@ -23,6 +23,15 @@
       end
     end
 
+    it "zipcode must be 5 digits" do
```

Comment:
Could we add one happy path assertion here ? `expect(correct_person).to be_valid`